newIDE/app/src/Utils/LocalFileDownloader.js

```diff
@@ -16,6 +16,21 @@ export type ItemResult<Item> = {|
   error?: Error,
 |};
 
+export const sanitizeUrlWithEncodedPath = (url: string) => {
+  // Split the URL into protocol and the rest of the URL

```

Comment:
Out of caution, I would check that if `://` is not found (`=== -1`), then you return the url as is (because it's probably not a URL, so better don't change it)